# Notes for OpenCyto Paper

## Intracellular Cytokine Staining

[What is ICS?](http://www.proimmune.com/ecommerce/page.php?page=ics) Although
this is an excellent definition, we need a citation for the paper. Perhaps a
good paper to cite would be Maecker et al. (2005) given below. See the first
quote. Also, see Nomura et al. (2008) below.

> Production of cytokines plays an important role in the immune
  response. Cytokines are involved in many different pathways including the
  induction of many anti-viral proteins by IFN gamma, the induction of T cell
  proliferation by IL-2 and the inhibition of viral gene expression and
  replication by TNF alpha. Cytokines are not preformed factors but are rapidly
  produced and secreted in response to cellular activation.

> Intracellular cytokine staining (ICS) is a very useful and widely used flow
  cytometry based assay which detects the production and accumulation of
  cytokines within the endoplasmic reticulum after cell stimulation.  ICS can be
  used in combination with other flow cytometry protocols for immunophenotyping
  using cell surface markers or with MHC multimers to detect an antigen specific
  response, making it an extremely flexible and versatile method.

## HVTN 065 Data Set

A paper on the HIV Vaccine Trials Network (HVTN) 065 data set was published by
Goepfert et al. (2011). The goal here is to summarize the relevant information
from the paper to incorporate into the OpenCyto paper.

Some terminology in terms of previous efficacy clinical trials:

> The next vaccine to complete an efficacy trial was MRKAD5, a trivalent
  adenovirus 5 (Ad5) vectored vaccine. This vaccine encoded Gag, Pol, and Nef
  and elicited HIVspecific CD8+ and CD4+ T cell responses in 75% and 41% of
  participants, respectively [2], but also failed to protect [3]. The most
  recent efficacy trial (RV144) tested 4 vaccinations of the vCP1521 canary pox
  vector, which encoded Gag, protease, and Env, with AIDSVAX B/E added to the
  last 2 vaccinations for a protein boost [4]. Although this combination vaccine
  did not induce measurable CD8+ T cells, it did induce antibody and CD4+ T
  cells and provided a 31% protection rate against infection.

Brief description of the HVTN 065 Phase 1 clinical trial:

> The National Institute of Allergy and Infectious Diseases HIV Vaccine Trials
  Network (HVTN) conducted a Phase 1 evaluation of the safety and immunogenicity
  of the pGA2/JS7 DNA and MVA/HIV62 vaccines in healthy, HIV-1 –seronegative
  adults. MVA62, given alone or with a JS7 DNA priming series, was safe and well
  tolerated, inducing T cell and antibody responses, which varied depending on
  the vaccine regimen, dose, and dosing schedule.

Study Design:

> HVTN protocol 065 was a randomized, double-blind, placebo-controlled trial
  conducted at 6 clinical sites in the United States (Figure 1B). Adults aged
  18–49 years who were deemed healthy based on medical history, physical exam,
  laboratory tests, troponin levels, and electrocardiogram (EKG) were enrolled.

Immune Response Assays:

> HIV-specific T cell responses were measured using intracellular cytokine
  staining conducted at HVTN central laboratories [2, 14]. Global potential T
  cell epitope [15] peptide pools representing HIV Env (3 pools), Gag (2 pools),
  and Pol (3 pools) were used at the final concentration of 1 lg for each
  peptide per milliliter. Cells were first stained with the Violet Live/Dead
  Fixable Dead Cell Stain [16] and then fixed, permeabilized, and stained with
  the following reagents: CD3-ECD, CD4-FITC, CD8-PerCP-Cy5.5, IFN-g-PE-Cy7,
  IL-2-PE, TNF-a-Alexa 700, and IL-4-APC. Positive responses were identified
  using the 1-sided Fisher exact test to support comparison of differences
  between background measurements and the numbers of CD4+ or CD8+ T cells
  producing IFN-g and/or IL-2 in response to peptide stimulation [14]. The
  breadth and depth of T cell responses [17] were calculated by the number of
  peptide pools eliciting a positive reaction per vaccinee (8 maximum).

Participant information:

> The median age of participants was 24 years, and 58% were female. The majority
  were white (73%) or African American (16%). All 120 participants received
  their initial vaccine, and 104 (87%) received all prescribed doses.

HIV-1-specific T Cell Responses:

> HIV-1–specific T cell responses were readily detected in all groups; however,
  the response rates depended on the vaccine regimen (Figure 3). The DNA prime
  enhanced both CD4 and CD8 response rates, with 2 DNA primes (either 1/10th or
  full dose) being more effective than a single full-dose DNA prime.
  Vaccine-induced CD4+ T cells were measured in 88% of individuals vaccinated
  with the 1/10th-dose and 77% of those vaccinated with the full-dose DDMM
  regimen. This compared to peak CD4+ response rates of 50% for DMM and 43% for
  MMM regimens (Figure 3A). Peak CD8+ T cell responses were 33% in the
  participants receiving 1/10th-dose and 42% in participants receiving the
  full-dose DDMM regimens compared to 22% and 17% in participants in the DMM and
  MMM regimens, respectively (Figure 3B). The magnitudes of responses were
  overlapping for all groups, with medians between .07 and .17% of total CD4+ T
  cells and between .06 and .65% of total CD8+ T cells (Figures 4 and 5A).

> At 6 months following the final vaccination, CD4+ T cell response rates were
  38% for DDMM (49% of their 2-week peak) compared with 8% for MMM (19% of their
  2-week peak) ($P < .03$); and CD8+ T cell response rates were 38% for DDMM
  (90% of their peak) compared with 4% for MMM (24% of their peak) (Figure
  4). For the DDMM and MMM regimens, the functionality of T cell responses, as
  measured by coproduction of IFN-g, IL-2, and TNF-a, were similar except for
  different patterns for single-cytokine-producing cells (Figures 4C and
  4D). Among the single-cytokine-producing CD4+ cells, IL-2 predominated in the
  DDMM group and TNF-a, in the MMM group. Among single–cytokine-producing CD8+ T
  cells, IFN-g production was most frequent in the DDMM group, whereas no single
  cytokine dominated in the MMM group. For both regimens, approximately
  one-third of the responding cells produced 3, 2, or 1 cytokine (Figures 4E and
  4F).

> The breadth and depth [17] of T cell responses against 2 Gag, 3 Env, and 3 Pol
  peptide pools revealed responses primarily directed to Gag and Env (Figure
  5A).

> Responses for CD4+ and CD8+ T cells are for responses to Gag, Env, or Pol
  measured as IFN-g– or IL-2–producing cells scored using intracellular cytokine
  staining (ICS) following stimulation with potential T cell epitope peptide
  pools.



> The kinetics of T cell responses differed for Gag and Env: following the final
  MVA dose, CD8+ T cell response rates for Gag increased 9-fold in the DDMM and
  4-fold in the MMM group; whereas CD8+ responses for Env increased 2-fold for
  DDMM recipients and decreased by 3-fold for MMM recipients. The DNA prime
  biased the response toward CD4+ T cells. This bias was strongest after the
  first MVA boost, when it ranged from a 7-fold to a 14- fold excess of CD4+
  responses over CD8+ responses in the DNA-primed groups compared to a 2.4-fold
  excess of CD4+ over CD8+ responses in the MMM group.

HIV-1–specific Antibody Responses:

> In contrast to T cell responses, HIV-1 –specific antibodies were induced more
  frequently and at higher levels by the full-dose than the 1/10th-dose DDMM
  regimen, and the highest frequencies and titers of Ab responses were induced
  by the MMM regimen (Figures 3C and 6). After the final vaccine administration,
  nearly all MMM recipients (96.6%) tested positive by the Abbott HIV-1/HIV-2
  ELISA, whereas only 73% of the DDMM recipients had seroconverted by this test
  ($P < .03$). A trend toward an increased number of positive Western blot
  assays was also seen in the MMM (21.4%) versus the full-dose DDMM (4.3%)
  recipients.

> Env-specific antibodies, as measured by binding to an immunodominant Ab for a
  gp120 monomer of the ADA vaccine Env (SP400 gp41 peptide), and neutralizing
  activity for HIV1MN, were all highest in the MMM group (Figure 6).
  Participants receiving the full-dose DDMM regimen had the lowest Ab responses,
  and the DMM group had intermediate antibody responses. The higher titers of Ab
  present in the MMM group were associated with this group receiving 3 doses of
  MVA. Following 2 doses of MVA, Ab responses were overall similar in the DDMM
  and DMM groups compared with those in the MMM group after 2 doses of MVA
  (Figure 6).

Vector-induced Immune Responses:

> The CD4+ and CD8+ T cell responses specific to the MVA vector were assessed
  following the MVA62 boosts. Subsequent to the first MVA boost, both the
  response rates and magnitudes of vaccinia-specific CD4+ and CD8+ T cells were
  significantly lower in those receiving the DNA prime compared with those
  receiving only MVA62 ($P < .001$) (Figure 7).

Discussion:

> However, the DDMM regimen induced the highest frequency and most persistent
  CD4+ and CD8+ T cell responses. The DNA prime also increased the breadth and
  depth of the T cell responses and biased these responses toward CD4+ T
  cells. On the other hand, the MMM regimen induced the highest frequency and
  magnitude of antibody responses. The single DNA prime (DMM) induced
  intermediate T cell and Ab responses. The 1/10th-dose DDMM regimen decreased
  the height of Ab responses but had limited effect on T cell responses.

> Thus T cell responses, below the level of detection by ex vivo analysis, were
  able to focus T cells to HIV antigens expressed by MVA62. Moreover, the
  absence of detectable DNA-elicited responses did not preclude the DNA prime
  curtailing anti-MVA T cell responses. Presumably this effect was due to the
  DNA prime establishing memory T cells recognizing HIV antigens that competed
  for antigen presentation during the boost with naive T cells recognizing MVA
  or HIV antigens. For all regimens, the number of MVA boosts was important for
  increasing both anti-Gag CD8+ T cell and anti-Env Ab responses.


## Relevant Papers and Points to Consider

### Excerpt from Introduction of flowCore's Vignette

Traditionally, flow cytometry has been a tube-based technique limited to
small-scale laboratory and clinical studies. High throughput methods for flow
cytometry have recently been developed for drug discovery and advanced research
methods (CITE: Gasparetto et al., 2004). As an example, the flow cytometry high
content screening (FC-HCS) can process up to a thousand samples daily at a
single workstation, and the results have been equivalent or superior to
traditional manual multi-parameter staining and analysis techniques.

The amount of information generated by high throughput technologies such as
FC-HCS need to be transformed into executive summaries (which are brief enough)
for creative studies by a human researcher (Brazma, 2001). Standardization is
critical when developing new high throughput technologies and their associated
information services (Brazma, 2001; Chicurel, 2002; Boguski and McIntosh,
2003). Standardization eorts have been made in clinical cell analysis by ow
cytometry (Keeney et al., 2004), however data interpretation has not been
standardized for even low throughput FCM. It is one of the most difficult and
time consuming aspects of the entire analytical process as well as a primary
source of variation in clinical tests, and investigators have traditionally
relied on intuition rather than standardized statistical inference (Bagwell,
2004; Braylan, 2004; Parks, 1997; Suni et al., 2003).

### Tarnok (2007)

> Automated pattern recognition is a prerequisite of cytometric high-throughput
  and high-content imaging. Whereas performing cell and cell-structure
  recognition on microscopic images is an art its automation initiates
  scientific analysis. Only by automated recognition will quantitative cell and
  tissue analysis become operator unbiased and an objective tool for research
  and diagnosis. It is also a prerequisite for high-throughput and high-content
  analysis of cell cultures, tissues and, probably in the future, of in vivo
  imaging.

### McNeil et al. (2013)

The authors analyzed the results of 110 international participating labs of ICS
data from the Cancer Immunotherapy Consortium on 9 FCS files. The authors
concluded that different gating strategies between labs can results in a large
source of variability in reported results. In the discussion, the authors
suggest that people with the know-how:

* should help us identify specific software and algorithms that may be used to
  more precisely identify and quantify rare antigen-specific cells of interest.

* Also, current tools are not adequate for the types of assays we are running or
  too complex for the average user.

With this mind, software tools need to be:

* Easy to use

* Amenable to differences in experimental conditions

* Less computationally intense

* Readily available to the entire flow-cytometry community

### Maecker et al. (2012)

The following quote is from the introduction. This could be useful to
incorporate into the introduction for the OpenCyto paper. Also, it has several
references that we should consider for inclusion.

> Flow cytometry has increasingly become a tool of choice for the analysis of
  cellular phenotype and function in the immune system. It is arguably the most
  powerful technology available for probing human immune phenotypes, because it
  measures multiple parameters on many individual cells. Flow cytometry thus
  allows for the characterization of many subsets of cells, including rare
  subsets, in a complex mixture such as blood. And because of the wide array of
  antibody reagents and protocols available, flow cytometry can be used to
  assess not only the expression of cell-surface proteins, but also that of
  intracellular phosphoproteins and cytokines, as well as other functional
  readouts.

> Microarray data are less complex than flow cytometry data, which are based on
  many hierarchical gates. Still, a reasonable degree of standardization of flow
  cytometry assays should be possible to achieve.

Useful definition of **gates**:

> Sequential filters that are applied to a set of flow cytometry data to focus
  the analysis on particular cell subsets of interest

Very important discussion of variability induced by data analysis. Nima's paper
is cited as well as Maecker et al. (2005).

> Data analysis is one of the largest variables in flow cytometry, as shown by
  studies analysing data at multiple sites versus centrally. It is also one of
  the easiest variables to address, as re-analysis of existing data is always
  possible, whereas choices made about sample handling and instrument setup are
  irrevocable. Central analysis is simple in concept, but it can be overwhelming
  in terms of the demands placed on one or a few coordinated personnel who must
  review all of the data to achieve consistent gating. Fortunately, automated
  gating algorithms have proliferated and improved, such that they now compete
  favourably with expert manual gating. A group of interested experts (FlowCAP),
  with support from the NIH, has completed two workshops aimed at comparing and
  improving the use of automated gating algorithms for flow cytometry
  analyses. Their continued work in this area should lead to standard software
  packages that can carry out easy and unbiased analyses of flow cytometry
  files. It is clear that individual solutions are available in each of the areas
  that lead to flow cytometry variability. What is missing is: guidance for
  those designing clinical trials on the integrated use of these solutions;
  resources to enable the implementation of all appropriate solutions; and some
  standardized approaches to the most common flow cytometry immunophenotyping
  challenges. With regard to such standardized approaches, the preformatted
  antibody cocktails being produced by the HIPC might provide a good start. By
  predetermining the reagent choices and simplifying sample handling, they
  should alleviate much of the variability in these areas. Because they
  represent a single set of reagent cocktails, they are also amenable to
  standardized instrument setup and data analysis routines, which should further
  decrease variability. Their use by the HIPC and by other interested
  laboratories should produce an example body of data that could demonstrate to
  the field the benefits of a standardized approach to flow cytometry, and to
  immunological assays in general

Finally, a succinct, convincing definition of mass cytometry (CyTOF):

> Mass cytometry: Flow cytometry using antibodies tagged with heavy metal ions,
  which are detected by mass spectrometry (as opposed to classical flow
  cytometry, which uses antibodies tagged with fluorophores and optical
  detection).

A brief glimpse into a possible future of CyTOF. This may be important to
mention in the discussion section of the OpenCyto paper. OpenCyto is not limited
to flow-cytometry data but is poised to handle CyTOF data.

> Another new technology, mass cytometry (CyTOF, DVS Sciences), is poised to
  revolutionize flow cytometry by allowing many more antibody probes to be
  combined in a single assay, while simultaneously eliminating the issue of
  optical spillover. If reagents for this platform become standardized, the
  design of antibody panels would be much easier, because the inclusion of
  certain reagents would not influence the performance of other reagents (as it
  does for conventional flow cytometry).

Flow is primarily used in a research setting but with standardization will be
ready for the clinical setting. This would be a good argument for the motivation
of OpenCyto:

> With the major exception of CD4+ T cell counting for immune monitoring in
  patients with HIV and the immunophenotyping of leukaemias and lymphomas, flow
  cytometry is mostly carried out in a research setting.

### Maecker et al. (2010)

> There are still many factors that impede the widespread use of flow cytometry
  in clinical trials. These can be roughly categorized as issues of sample
  handling, instrument setup and data analysis.

> In the area of data analysis, there have been advances in gating tools and
  batch-analysis options. However, the analysis software now available does not
  allow efficient archiving and retrieval of large amounts of data or analysis
  across multiple experiments. The tools available are still highly focused on
  experiment-specific analysis and are generally insufficient to achieve the
  ultimate goal of reliable, single-step transformation of raw data into
  quantified results for large numbers of files.

> Perhaps the largest single contributor to variability in flow cytometry is
  differences in gating.

> Instrument setup and statistical counting errors accounted for only a very
  minor proportion of the variability, whereas individualized gating methods
  accounted for the vast majority of the interlaboratory variation.

> In the study described above, the inclusion of ‘dim’ populations for key
  markers such as CD4 and CD8 accounted for most of the gating variability
  noted. When populations are tightly clustered and easily discriminated from
  each other, such variability will of course be less. This means that a certain
  amount of gating variability can be avoided by optimal design of reagent
  panels. However, the remaining variability needs to be handled through the use
  of either a shared gating template or central analysis by a single operator.

I didn't understand exactly what MFI was previously. Now, I realize that it is
quite simple -- the median of channel conditional on the parent population.

> In cases in which a cell population displays a continuous distribution of
  staining intensity, rather than discrete positive and negative populations, it
  can be more appropriate to report the median fluorescence intensity of the
  entire cell population. However, differences in staining and instrument setup
  from experiment to experiment warrant the use of some type of standard to
  ensure the reproducibility of this approach. This could involve simply
  calculating the ratio of the median fluorescence for the experimental sample
  to that of a sample stained with isotype-matched control antibody.

> Variability due to subjective gating and how positive versus negative events
  are delineated can result in considerable deviations. At a minimum, templates
  for acquisition and data analysis would be distributed among the remote
  sites. Perhaps centralization of data analysis would be even more desirable.

### Maecker et al. (2005)

> Enzyme-linked immunospot (ELISPOT) and cytokine flow cytometry (CFC) (or more
  specifically, intracellular cytokine staining (ICS)) are popular methods for
  singlecell analysis of antigen-specific T cell cytokine production. T cell
  production of IFN-gamma, and increasingly also IL-2, is taken as a measure of
  vaccine immunogenicity in experimental vaccine trials. Of the two types of
  assays, ICS has the advantage of a highly multiparametric read-out (flow
  cytometry) that allows for precise phenotyping of the responding T cell
  populations.

The following quote cites Pantaleo and Koup (2004). This paper may be useful to
read and to cite in the OpenCyto paper when discussing correlates of vaccine
response.

> While most current HIV trials are not powered to determine efficacy, and
  cytokine production has not been validated as a surrogate marker of protection
  from HIV infection or progression, there is nevertheless a desire to measure
  immunogenicity of candidate vaccines as well as safety in early clinical
  trials.

Another quote that discusses variability in gating:

> A large proportion of the site-to-site variability could be explained by
  differences in gating of the ICS data.

### Aghaeepour et al. (2013)

According to the paper, automated gating methods have reached a sufficient level
of maturity and accuracy for reliable use in flow-cytometry data analysis.

The following quote cites Lugli et al. (2010).

> This analysis is usually accomplished by sequential manual partitioning
  ("gating") of cell events into populations through visual inspection of plots
  in one or two dimensions at a time. Yet many problems have been noted with
  this approach to FCM data analysis, including its subjective, time-consuming
  nature and the difficulty in effectively analyzing high-dimensional data.

The following quote cites 23 different papers on the surge of automated methods
proposed. We may want to cite some of the same. These include the description of
the flowClust model and the FLAME paper from Pyne et al. (2009).

> Since 2007, there has been a surge in the development and application of
  computational methods to FCM data in an effort to overcome these serious
  limitations in manual gating–based analysis, with successful results reported
  in each case.

> Automated methods perform very well in sample classification, even for data
  sets that were challenging for manual analysis.

> The excellent performance of automated methods, even with the relatively
  challenging HVTN data set, was somewhat surprising but indicates that
  automated methods can perform well on sample classification use cases,
  detecting useful biomarkers in FCM data.

### Ge and Sealfon (2012) -- the flowPeaks paper

Nothing specific here. However, it does appear quite fast but very ad hoc. Given
that it's a recent paper and did fairly well in FlowCAP, this paper should be
cited as one of the modern gating algorithms proposed.

### Lugli et al. (2010)

When we discuss polyfunctionality of cytokines for classification, it would be
useful to cite this paper as well as Betts et al. (2006) as mentioned in this
quote:

> Betts et al. used polychromatic flow cytometry to analyze the simultaneous
  production of IFN-g, TNF-a, IL-2, MIP-1b, and the degranulation marker CD107a
  and showed that highly polyfunctional HIV-specific CD8+ T cells (i.e., cells
  producing four or five functional markers simultaneously) were more frequent
  in HIV long-term nonprogressors than in progressors.

Furthermore, the **polyfunctional** term was coined by Harari et al. (2005). The
current paper briefly describes why multiple cytokines are considered. We should
at least pay homage to Harari et al. (2005) when we describe the classification
study. Here's a related quote:

> Originally, when intracellular cytokine staining (ICS) technique was first
  introduced, researchers measured only one function, i.e., IFN-g, as a
  surrogate marker for any T cell response. IFN-g was thought to be the dominant
  immune function of the effector T cell response and many candidate vaccines
  moved to phase I–II because of their capability to induce a strong production
  of this cytokine. However, simultaneous staining of two cytokines at the same
  time, e.g., IFN-g and IL-2, revealed that T cell are heterogeneous, as single
  cytokine-producing cells as well as double producers are present (Harari,
  2005). The authors coined the word ‘‘polyfunctional’’ as referred to cells
  that were able to produce both cytokines at the same time. The first
  assessment of five cytokines simultaneously revealed that both CD4+ and CD8+ T
  cells were even more heterogeneous and multiple subsets of T cells with
  different functionality could be identified according to the
  cytokine-producing potential (De Rosa, 2004). From here the idea to brake-down
  the immune response by identifying all combinations of cytokine-producing T
  cells.

Yet another excellent quote that we should somehow incorporate into the
motivation for OpenCyto:

> Certainly, the future in this field is the use of automatic tools of data
  analysis that can identify cell populations and possibly underline their
  relative importance. We believe that manual gating strategies will be largely
  reconsidered and modified over the next few years as robust and reliable
  methods of automated gating are being proposed. In particular, unsupervised
  approaches applied to raw data, such as CA and PCA, will become more and more
  popular, as they consider the fluorescence values of each single cells and do
  not presume the previous partition of the data, e.g., distinction between
  positive and negative expression. The main goal of these automatic tools of
  analysis will become the identification of similarities and differences among
  samples. Certainly, an algorithm cannot substitute the expertise of the
  operator in data analysis, especially when specific populations of interest
  need to be identified. However, different applications such as high-throughput
  and high-content screening assays for compounds nowadays require the fast
  extrapolation of information.

> More and more markers are now combined to better characterize malignant cells;
  automated analytical tools applied to multiparameter dataset will allow us to
  better define the type of disease, its stage, and the progression rate.
  Computers and algorithms combined with advanced operator expertise and
  instrument standardization will certainly improve our knowledge in basic and
  translational research.

### Bashashati and Brinkman (2009)

Yet another quote criticizing manual gating:

> In practice, FCM data analysis is performed manually, a process that requires
  an inordinate amount of time and is error-prone, nonreproducible,
  nonstandardized, and not open for re-evaluation, making it the most limiting
  aspect of this technology.

The introduction in this paper is short, but it has several excellent points that we should make, including:

> The increased volume and complexity of FCM data resulting from the increased
  throughput greatly boosts the demand for reliable statistical methods and
  accompanying software implementations, for the analysis of these data

> Interpretation of gating results is highly dependent on what the objective of
  the study is. Usually, there are two major objectives in an FCM-based study:
  (a) statistical comparison of samples, where the samples are compared to see
  if they share similar characteristics; (b) classification, where the samples
  are labeled to predefined classes such as healthy versus patient or patients
  with short survival versus the ones with long survival time.

A brief mention about the difficulty of gating rare populations:

> Sometimes it is possible that even if the actual number of clusters is known,
  the clustering algorithm may not identify the correct clusters at the level of
  separation that is desired. This can happen when there is a rare cell
  population within the FCM data. In this case, the clustering algorithm may
  consider the rare population as an outlier or as part of a larger cell
  population and instead divide larger cell populations into smaller
  populations. One approach to overcome this problem might be clustering the
  data with higher number of clusters with the hope that the rare populations
  are represented by separate clusters and use some merging algorithm to combine
  the clusters that are similar according to a criterion.

Here is an excerpt of the conclusion. Notice the call to look beyond automated
gating and to automated pipelines:

> To date, most research has been focused on developing approaches for automated
  gating of FCM data. Manual gating is recognized as labor intensive,
  subjective, and prone to error when processing large numbers of samples.
  Therefore, automated gating methods will allow for a faster and more robust
  data analysis pipeline. Although significant effort is still needed to develop
  automated gating algorithms that address challenging aspects of FCM data, we
  believe that the research community needs to look beyond automated gating and
  develop bioinformatics tools that facilitate building completely automated FCM
  data analysis pipelines. It should be noted that the development of robust,
  automated methods for high-throughput FCM data analysis also requires
  high-quality data to feed into the analysis framework. Generating this
  high-quality data requires well-designed experiments with the appropriate
  positive and negative controls. A rigorous quantitative assessment is
  important before using automated approaches in practice, as a replacement for
  expert manual analysis. Moreover, it is likely that one data analysis solution
  may not be suitable to address specific questions of a study or address the
  challenges of analyzing a specific FCM dataset. For example, if somebody is
  interested in identifying a previously known type of cell, supervised
  techniques might be better suited. Overall, in order to use automated data
  analysis approaches in biomedical research and clinical setting, we need to
  develop more generic solutions or design smart algorithms that can tune
  themselves with little intervention, as the users may not have enough
  knowledge of bioinformatics techniques.

### Bocsi and Tarnok (2008)

In the OpenCyto paper, we need to mention that at times, a poor sample or an
artifact of the gating algorithm can yield a poor gate. We need to stress the QC
feature Mike developed and cite this quote:

> Full automation is untenable in the high-throughput paradigm, and research on
  automating analysis of FCM data is ongoing. However, any automated method,
  however efficient, is bound to fail on occasion. Thus, it is important to have
  diagnostic tools that can quickly identify such failures so that they can be
  dealt with appropriately.

### Aghaeepour et al. (2010)

> However, data analysis can be a significant challenge because each data set is
  a multiparametric description of millions of individual cells. Consequently,
  despite widespread use, FCM has not reached its full potential due to the lack
  of an automated analysis platform to assist high-throughput data generation.

The second paragraph of this paper provides an excellent model to follow when
summarizing previous work on automated gating.

> A critical bottleneck in data analysis is the identification of groups of
  similar cells for further study (i.e., gating). This process involves
  identification of regions in multivariate space containing homogeneous cell
  populations of interest. Generally, gating has been performed manually by
  expert users, but manual gating is subject to user variability (4–6) and is
  unsuitable for high-throughput data analysis (7). Several methods have been
  developed to automate the gating process (8). flowClust is a model-based
  clustering approach that models cell populations using a Box-Cox
  transformation to remove skewness followed by a mixtures of t-distributions
  (9). flowMerge (10) extends the flowClust framework by applying a cluster
  merging algorithm (11) to allow multiple components to model the same
  populations, enabling it to fit concave cell populations. FLAME (12) uses a
  mixture of skew-t-distributions to make the model more flexible to skewed cell
  populations. curvHDR (13) is a nonparametric density-based approach, and
  therefore is not limited to identifying cell populations based on
  shape. curvHDR models cell populations based on the curvature of the
  underlying distribution. However, it requires user-defined parameter values
  and cannot be applied to more than three dimensional data. SamSpectral (14)
  uses an spectral clustering algorithm to find cell populations, including
  nonconvex ones. Given the high time and memory requirements of the spectral
  clustering algorithm, SamSpectral finds cell populations based on
  representative subsampling of the data; however, this can potentially decrease
  the quality of the gating as some biological information can be lost during
  the sampling process. SamSpectral also requires user-defined parameter values
  for each data set of similar experiments.

### Pyne et al. (2009)

Specific quote about manual gating not being reproducible.

> The operator subjectively demarcates a cell population while moving through
  successive 2- or 3-dimensional projections of the data. This process limits
  the reproducibility of data processing.

Other than this quote, this paper is yet another gating algorithm proposed that
we should cite.

### Chan et al. (2008)

Although this paper has some very useful advice about mixture modeling (e.g.,
adding an extra mixture components when cells pile up against the y-axis), for
the OpenCyto manuscript, the current paper is yet another gating algorithm
proposed that we should cite.

### Nomura et al. (2008)

Another excellent quote that explains what ICS is:

> Cytokine flow cytometry (CFC), also known as intracellular cytokine staining
  (ICS), is a technique that facilitates the detection of cellular responses,
  which are not measurable on the cell surface. It enables the detection of rare
  antigen-specific cell populations, so that T-cell responses can be quantitated
  without the need for *in vitro* expansion.

### De Rosa et al. (2004)

> We demonstrate in this study for the first time the simultaneous analysis of
  five cytokines at the single-cell level for Ag-specific T cells. Our results
  show that T cells are capable of expressing diverse cytokine profiles, and
  thereby demonstrate that classifying T cells into a small number of
  cytokine-based functional subsets is probably inadequate to characterize
  immune responses.

> These results demonstrate that no single functional parameter adequately
  quantifies responses, and in fact, due to interindividual variability, limited
  combinations of functional parameters will not be adequate for all
  individuals. Therefore, optimal measurements of immunogenicity of a vaccine or
  of immune responses to pathogenesis will require the measurement of multiple
  distinct functional outcomes.

> The methods introduced in this report provide a novel means for monitoring
  vaccine-induced immunogenicity and for monitoring T cell responses to
  infection. In terms of vaccine development, a method for the identification of
  those T cell responses induced by candidate vaccines that correlate with
  efficacy will help target the appropriate vaccines for further clinical
  testing.

> In summary, we found that in the human, T cell responses to immunization are
  broad and complex, encompassing a far wider spectrum than can be measured
  solely by the production of a single cytokine. It is evident that sensitive
  quantification of immune responses requires the coordinate measurement of more
  than a single response, and that IFN-g is a subdominant effector response in
  many immunizations.